src/main/java/uk/gov/hmcts/reform/sscs/models/refdata/RegionalProcessingCenter.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.sscs.models.refdata;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
```

Comment:
@sathishbabuj I refactor this class to make it immutable. It's a good practice to follow because this way you make it thread safe, etc. 

More details here: 
https://goo.gl/JWTSAb
http://www.informit.com/articles/article.aspx?p=1216151&seqNum=2